src/apiCommands.js

```diff
@@ -0,0 +1,42 @@
+// set the name of the game
+
+// fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games', {
```

Comment:
- [ ] [OPTIONAL] i I kindly suggest that you remove commented out code from your codebase as it doesn't make your work look neat